diff --git a/editors/scel/el/CMakeLists.txt b/editors/scel/el/CMakeLists.txt
index dbc0a5f..9731a28 100644
--- a/editors/scel/el/CMakeLists.txt
+++ b/editors/scel/el/CMakeLists.txt
@@ -36,6 +36,20 @@ if (SC_EL_BYTECOMPILE)
 
 		install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${el}c
 		DESTINATION share/emacs/site-lisp/SuperCollider)
+
+        add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/${el}.gz
+            COMMAND /usr/bin/gzip -9n -c --
+                ${CMAKE_CURRENT_BINARY_DIR}/${el} >
+                ${CMAKE_CURRENT_BINARY_DIR}/${el}.gz
+            DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/${el}
+            WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+            COMMENT "Creating gzipped sources")
+
+        add_custom_target(${el}.gz ALL
+		DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/${el}.gz)
+
+        install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${el}.gz
+	    DESTINATION share/emacs/site-lisp/SuperCollider)
 	endforeach()
 else()
 	install (FILES ${scel_sources} ${CMAKE_CURRENT_BINARY_DIR}/sclang-vars.el
